OpenRA.Mods.D2k/Traits/World/WormManager.cs

```diff
@@ -22,16 +22,17 @@ namespace OpenRA.Mods.D2k.Traits
 	class WormManagerInfo : ITraitInfo
 	{
 		[Desc("Minimum number of worms")]
-		public readonly int Minimum = 2;
+		public readonly int Minimum = 1;
 
 		[Desc("Maximum number of worms")]
 		public readonly int Maximum = 4;
 
-		[Desc("Average time (seconds) between worm spawn")]
-		public readonly int SpawnInterval = 120;
+		[Desc("Average time (in ticks) between worm spawn")]
+		public readonly int SpawnInterval = 3000;   // Approximately 2 minutes
```

Comment:
While that is technically true there is no need to add complexity where it doesn't need to exist.

I also just noticed the wording "average time" here while this is in fact a set timeframe, not an average.